validation/resource-config_test.go

```diff
@@ -36,7 +36,7 @@ func TestResourceConfig(t *testing.T) {
 		actualResourceConfig, err := getResourceConfigWithRetries(context.Background(), common.Address(contractAddress), client)
 		require.NoErrorf(t, err, "RPC endpoint %s: %s", rpcEndpoint)
 
-		require.Equal(t, bindings.ResourceMeteringResourceConfig(StandardConfig.ResourceConfig), actualResourceConfig, "resource config unacceptable")
+		require.Equal(t, bindings.ResourceMeteringResourceConfig(StandardConfigMainnet.ResourceConfig), actualResourceConfig, "resource config unacceptable")
```

Comment:
Why do we hardcode mainnet here? Instead should we have a single `StandardConfig` loaded in init and passed into tests, where the data loaded is based on the L1 chain ID? (or similar deduping instead of repeating the switch-case statements I see in gpo-params-test and l2oo_test)

Reply:
I think for these particular parameters there has not yet been a "need" to separate out by superchain target, so all standard chains are checked against the mainnet expectations.

I think you are right though, we should be loading in the appropriate set of parameters (which exist in both TOML files). I can clean this up.

Reply:
[refactor StandardConfig to be a mapping keyed by superchain name](https://github.com/ethereum-optimism/superchain-registry/pull/188/commits/66e8b48c65821d197a3d49570a0502c2e001d20a)